### ISSUE TITLE
CI check that tutorials render

### ISSUE THREAD

**Issue by dylanhmorris** [CDCgov/pyrenew-hew#191]
Eventually could become a website 

> Would have been caught if we had tutorials run in CI :(

_Originally posted by @damonbayer in https://github.com/CDCgov/pyrenew-hew/pull/186#discussion_r1871969606_

**Comment by SamuelBrand1** [comment]
Part of a meta-issue about lack of CI running different aspects I think.

**Comment by damonbayer** [comment]
I don't think we plan to put these on a website, but maybe the rendered tutorials can be part of a branch.

**Comment by damonbayer** [comment]
Tutorials have been removed.
